frontend/src/pages/ActivityReport/Pages/Review/Submitter/index.js

```diff
@@ -94,18 +94,23 @@ const Submitter = ({
   const filtered = pages.filter((p) => !(p.state === 'Complete' || p.review));
   const incompletePages = filtered.map((f) => f.label);
 
+  /*
+  grantsMissingMonitoring:
+  * This checks that if we only have a single monitoring goal selected,
+  * that all selected recipient/grants are associated with that goal.
+  * If not they either need to remove the recipient or add a standard goal.
+  */
   const grantsMissingMonitoring = () => {
```

Comment:
These functions are pretty complicated/a lot of nested stuff happening. It's not a requirement for my approving this PR but we should consider refactoring so that they are a little more readable. Maybe even moving them to their own file.